configure.ac

```diff
@@ -242,7 +242,7 @@ AC_CONFIG_FILES([arpack$LIBSUFFIX.pc:arpack.pc.in], [], [LIBSUFFIX="$LIBSUFFIX"]
 AC_SUBST([CMAKE_INSTALL_PREFIX], ["$prefix"])
 AC_SUBST([BLAS_LIBRARIES], ["$BLAS_LIBS"])
 AC_SUBST([LAPACK_LIBRARIES], ["$LAPACK_LIBS"])
-AC_SUBST([EIGEN3_INCLUDE_DIR], ["$EIGEN3_CFLAGS"])
+AC_SUBST([EIGEN3_INCLUDE_DIR], ["${EIGEN3_CFLAGS/-I/}"])
```

Comment:
@turboencabulator : do you think there is a more portable / autotools way to do this ?

Reply:
@fghoussen That's a bashism, won't work on all platforms (particularly BSD, they've already had plenty of problems with bash).  I think this and #216 need to be reverted, they're going to cause more problems than they solve.  If I understand #214 right, underlinking is either the goal or the problem, and this will force overlinking.

Reply:
@turboencabulator : thanks for the feedback.
- if you think some patchs/bashism need to be reverted, could you PR some replacement ?
  At my side on debian/testing + bash #216 was indeed needed to compile/link. 
- Would really be good to add Free BSD jobs in CI...
  Too bad that docker can't make it (different kernels as far as I understand)

Reply:
@fghoussen Yeah I'll send a PR.  Can you elaborate on your testing?  Were you trying to use a static or a shared lib?  Ideally the output from pkg-config looks something like this, and if you're using a static lib then it'll give you the extra flags you need to link (making this whole change unnecessary).
```
$ pkg-config --libs arpack
-larpack
$ pkg-config --libs --static arpack
-larpack -llapack -lblas
```

Reply:
@turboencabulator : thanks. Tried both linking.